buildscripts/conda_recipes/pixie/meta.yaml

```
@@ -1,37 +1,42 @@
-{% set name = "pixie" %}
-{% set VERSION_SUFFIX = "" %} # debug version suffix, appended to the version
+{% set __NAME__ = "pixie" %}
+{% set __VERSION_SUFFIX__ = "" %} # debug version suffix, appended to the version
+{% set __VERSION__ =  "%s%s" % (environ.get('GIT_DESCRIBE_TAG', ''), __VERSION_SUFFIX__) %}
 
 package:
-   name: {{ name }}
-   version: {{ "%s%s" % (environ.get('GIT_DESCRIBE_TAG', ''), VERSION_SUFFIX) }}
+   name: {{ __NAME__ }}
+   version: {{ __VERSION__ }}
 
 source:
    path: ../../..
 
+
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install -vv --no-deps --ignore-installed .
```

```suggestion
  script: {{ PYTHON }} -m pip install -vv --no-deps --no-build-isolation --ignore-installed .
```
`--no-build-isolation` is a default flag currently

---

done in [9c8f98a](https://github.com/numba/pixie/pull/20/commits/9c8f98acfd3fa727d71832222cb2eea6dfd799e4)